nix/cardano-bridge.nix

```diff
@@ -12,11 +12,11 @@ in runCommand "daedalus-cardano-bridge" {
   mkdir -pv $out/bin
   cd $out/bin
   echo ${cardano-wallet.version} > $out/version
-  cp ${cardano-wallet.cardano-wallet-byron}/bin/cardano-wallet-byron* .
-  cp ${cardano-shell.nix-tools.cexes.cardano-launcher.cardano-launcher}/bin/cardano-launcher* .
-  cp ${cardano-node}/bin/cardano-node* .
-  cp ${export-wallets}/bin/export-wallets* .
-  cp ${cardano-cli}/bin/cardano-cli* .
+  cp ${cardano-wallet.cardano-wallet-byron}/bin/* .
+  cp -f ${cardano-shell.nix-tools.cexes.cardano-launcher.cardano-launcher}/bin/cardano-launcher* .
+  cp -f ${cardano-node}/bin/cardano-node* .
+  cp -f ${export-wallets}/bin/export-wallets* .
+  cp -f ${cardano-cli}/bin/cardano-cli* .
   ${pkgs.lib.optionalString (target == "x86_64-windows") ''
     cp ${pkgsCross.libffi}/bin/libffi-6.dll .
```

Comment:
This will also need `cp -f`.

Reply:
@rvl fixed in https://github.com/input-output-hk/daedalus/pull/2004/commits/d0430be89e120b4687d9b771804b039166c1c9ae